src/tree/containers/ContainersTreeItem.ts

```diff
@@ -75,9 +78,22 @@ export class ContainersTreeItem extends LocalRootTreeItemBase<ILocalContainerInf
         }
     }
 
-    public getTreeItemForEmptyList(): AzExtTreeItem[] {
-        const dockerTutorialTreeItem = new OpenUrlTreeItem(this, localize('vscode-docker.tree.conatiner.gettingStarted', 'Get started with Docker containers...'), 'https://aka.ms/getstartedwithdocker');
-        dockerTutorialTreeItem.iconPath = getThemedIconPath('docker')
-        return [dockerTutorialTreeItem];
+    protected getTreeItemForEmptyList(): AzExtTreeItem[] {
+        if (this.isNewContainerUser()) {
+            const dockerTutorialTreeItem = new OpenUrlAzExtTreeItem(this, localize('vscode-docker.tree.conatiner.gettingStarted', 'Get started with Docker containers...'), 'https://aka.ms/getstartedwithdocker');
+            dockerTutorialTreeItem.iconPath = getThemedIconPath('docker')
+            return [dockerTutorialTreeItem];
+        }
+        return super.getTreeItemForEmptyList()
+    }
+
+    private isNewContainerUser(): boolean {
```

Comment:
Let's use a class `boolean | undefined` here and update it to false in `updateNewContainerUser`, so that we don't have to use the memento every time.